### ISSUE TITLE
Incorrectly reported unused constant

### ISSUE THREAD

**Issue by mchf** [YorickPeterse/ruby-lint#194]
This is follow up of https://github.com/YorickPeterse/ruby-lint/issues/191. See details there. 

The only change is that src files were moved into ```app``` subdir as proposed in https://github.com/YorickPeterse/ruby-lint/issues/191 respectively in http://code.yorickpeterse.com/ruby-lint/latest/file.configuration.html#directories

When running ```ruby-lint *``` in project's dir I still get
```
const_container.rb: warning: line 2, column 3: unused constant STRING_CONST
```

**Comment by mchf** [comment]
I guess that it can be caused by implementation of ```unused_variables.rb``` analysis. When the constant is not used in the class where defined (or file?) then it is reported as unused. So, the analysis should have 'per project' scope instead of 'per file'.
